employees/permissions.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import permissions
+from users.models import CustomUser
+
+
+class IsReportAuthor(permissions.BasePermission):
```

Comment:
As for now, I am not using any of these anywhere in the app.
Also, I believe permissions should be located in directory such as utils, since everyone might require them in future development.

Should I drop this commit altogether?

Reply:
These permissions should be in `users` app since every `Permission` is about the `CustomUser`.

Reply:
Should I drop it then?

Reply:
No, maybe add them as first commit in branch (base commit, at the bottom - it means not at the top, I mean... You know... Trunk of tree... base of the mountain.. anyway...) and in `users` app (beware conflicts with Martyna).

Reply:
Done